docs/program/connectivity.md

```diff
@@ -3,25 +3,22 @@ title: "Client Sessions and Robot Network Connectivity"
 linkTitle: "Network Connectivity Issues"
 weight: 20
 type: "docs"
-description: "Whan a robot loses its connection to the internet, all client sessions will timeout and end by default."
+description: "When a robot loses its connection to LAN or WAN, all client sessions will timeout and end by default."
 tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
 ---
 
-What happens when your robot loses its connection to the internet over a LAN or WAN and can no longer communicate with clients through [the Viam app](https://app.viam.com).
+A [client session](/program/apis/sessions/) uses the most efficient route to connect to your robot either through local LAN or WAN or the internet.
 
-## How does `viam-server` work if your robot loses its network connection?
+When a robot loses its connection to the internet but is still connected to a local LAN or WAN:
 
-When a robot loses its connection over LAN or WAN, it can no longer communicate with clients through [the Viam app](https://app.viam.com).
-When no heartbeat has been received over a timeout period, `viam-server` will end any current client [*sessions*](/program/apis/sessions/) on this robot.
+- Client sessions connected through the internet will timeout and end.
```

Comment:
(opt) For clarity I would suggest moving this down and specifying "different LAN or WAN" or "through the cloud" vs just "through the internet", but idk I think this is technically correct anyways?